ci-operator/jobs/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-kueue-migration-presubmits.yaml

```
@@ -0,0 +1,248 @@
+presubmits:
+  opendatahub-io/opendatahub-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^kueue-migration$
+    - ^kueue-migration-
```

I think CI is complaining about something here

---

probably remove the ending - or the line all together? 
```suggestion
    - ^kueue-migration
```